staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/server.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apiextensions-apiserver/pkg/cmd/server/options"
+	genericapiserver "k8s.io/apiserver/pkg/server"
+)
+
+func NewCommandStartCustomResourceDefinitionsServer(out, errOut io.Writer, stopCh <-chan struct{}) *cobra.Command {
```

Comment:
Can we rename `NewCommandStartCustomResourceDefinitionsServer` to a better name? like `NewCustomResourceDefinitionsServerCommand` or `NewAPIExtensionsServerCommand`

Reply:
NewServerCommand? The package gives the context.

Reply:
sgtm